stock_quant_unique_serial/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,8 @@
+When creating or updating a stock.quant record with serial management products ,
+the constraint checks if serial number already exists. If a duplicate is found, the user get a warning message and the record is not saved.
+
+Standard behavior fails to prevent the creation of multiple records
+with the same serial number when different owners are specified for the same product.
+
+This can lead to warnings in the Inventory Adjustment view, which may be not desirable.
+So adds the ability to prevent the creation of multiple records with the same serial number, even when different owners are specified for the same product.
```

Comment:
```suggestion
This module adds a constraint on stock.quant to make sure that no stock exists for the specified serial when stock is being created (e.g. purchase receipt is being processed) to avoid the duplicates.

Standard behavior fails to prevent the duplicates, for example, when there is stock for a serial with no owner, and new stock comes in with the same serial with an owner. This module intends to cater to this case.
```